iOS/ViewControllers/Courses/CertificatesListViewController.swift

```diff
@@ -40,18 +46,57 @@ class CertificatesListViewController: UITableViewController {
 
 }
 
-extension CertificatesListViewController { // TableViewDelegate
+extension CertificatesListViewController: CardListLayoutDelegate {
+
+    func collectionView(_ collectionView: UICollectionView,
+                        heightForCellAtIndexPath indexPath: IndexPath,
+                        withBoundingWidth boundingWidth: CGFloat) -> CGFloat {
+
+        let cardWidth = boundingWidth - 2 * 14
+        let boxHeight = cardWidth / 2 - 20
```

Comment:
Please set this to `120`. 
Fixed for now. When fully supporting dynamic type, we have to calculate the maximum height for all cards depending on the size of the text.